src/Microsoft.AspNetCore.Sockets.Client/SendMessage.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Sockets.Client
+{
+    public struct SendMessage
```

Comment:
Nit: Not sure I love the naming of SendMessage and Message. I would suggest renaming Message to ReceiveMessage, but I get that's potentially little confusing from the perspective of the server. Unfortunately this makes the client code a little less clear. Especially considering the client transports receive SendMessages.